python/cudf/cudf/core/series.py

```diff
@@ -5190,7 +5201,7 @@ def value_counts(
         dtype: int32
         """
         if bins is not None:
-            raise NotImplementedError("bins is not yet supported")
+            self = cudf.cut(self, bins, include_lowest=True)
```

Comment:
Can you avoid doing `self = ` here and instead assign it to another variable? 

Docstring needs to be updated since `bins` is now supported and would be good to have an example demonstrating the same.

Reply:
Makes sense, sounds good! I'll work on this